build.gradle

```diff
@@ -21,8 +21,8 @@ allprojects {
     apply plugin: 'cz.alenkacz.gradle.scalafmt'
 
     sourceCompatibility = 1.8
```

Comment:
I think we should have sourceCompatibility set to 1.11 as well since we're aiming for 1.11